src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandlerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
+
+class DeprecationErrorHandlerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @dataProvider provideRegisterParameters
+     */
+    public function testRegisterFalse($mode)
+    {
+        $this->assertNull(DeprecationErrorHandler::register($mode));
```

Comment:
this creates global side effects (changing the error handler)
I'd remove this test entirely: DeprecationErrorHandler can't be unit tested but only functionally tested, which is already the case (because we use it quite extensively on the travis matrix, and it works)